ai_edge_torch/convert/fx_passes/build_aten_composite_pass.py

```diff
@@ -213,6 +213,35 @@ def is_valid_padding(padding: list[int]):
   node.target = avg_pool2d
 
 
+@_register_composite_builder(torch.ops.aten.embedding.default)
+def _aten_embedding(gm: GraphModule, node: Node):
+  op = node.target
+  args_mapper = TorchOpArgumentsMapper(op)
+
+  def embedding(*args, **kwargs):
+    nonlocal op, args_mapper
+    full_kwargs = args_mapper.get_full_kwargs(args, kwargs)
+    _, embedding_dim = full_kwargs["weight"].size()
+    idx = full_kwargs["indices"]
+    idx = idx.type(torch.int)
+    B, T = idx.size()
+
+    idx = torch.reshape(idx, (B * T,))
```

Comment:
I think injecting operations as we are building a composite op is adding to the places where we are modifying the program. This breaks the concept that composites are only a way to annotate the program with high level information.

I think the right design is to annotate the graph with pytorch info (build an `aten.embedding` composite) in the pytorch layer and then convert it to what tflite can consume in the tflite converter. This PR is using the tflite kernel/converter knowledge in the pytorch side of the stack.

Reply:
the issue here is that we want to avoid adding `reshape`s to the pytorch layer. doing this as fx passes will hide the complexity from the user.